examples/org.eclipse.ui.examples.perspective/plugin.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="org.eclipse.ui.perspectives">
+      <perspective
+            class="org.eclipse.ui.examples.perspective.PerspectiveFactory"
+            icon="$nl$/icons/full/etool16/default_persp.png"
+            id="org.eclipse.ui.examples.perspective.perspective"
+            name="Example Perspective">
+      </perspective>
+   </extension>
+   <extension
+         point="org.eclipse.ui.perspectiveExtensions">
+      <perspectiveExtension
+            editorOnboardingImage="$nl$/icons/default_persp_grey.png"
+            editorOnboardingText="Open a file or drop files here to open them."
```

Comment:
This should be externalized.

Reply:
I have added them to plugin.properties like e.g. plugin o.e.ui does it with extension texts.
Unfortunately they are not correctly retrieved in target and hence not shown in ui.
When I use the externalize wizard it adds them to a new file bundle.properties.

What am I doing wrong here?

Reply:
Ah found it...
`Bundle-Localization: plugin` need to be added to Manifest.MF.

Now it works.